### PR TITLE
Changed for LinkedIn and Formstack

### PR DESCRIPTION
**Pull request by katheesR**
23 test cases are faliled out of 53.

The test case are failing because of the major API change in the same API version.

https://developer.linkedin.com/support/developer-program-transition

### CODE REVIEW

File: linkedin/linkedin-connector/linkedin-connector-1.0.0/org.wso2.carbon.connector/pom.xml [hunk 1]
```diff
@@ -2,18 +2,12 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```
Comment (prabathabey) [inline linkedin/linkedin-connector/linkedin-connector-1.0.0/org.wso2.carbon.connector/pom.xml]: We don't seem to have a license added to this particular file.

File: linkedin/linkedin-connector/linkedin-connector-1.0.0/org.wso2.carbon.connector/src/main/resources/linkedin_people/getConnections.xml [hunk 2]
```diff
@@ -35,7 +35,7 @@
 			if (id != null && id != "") {
 				query = apiUrl + 'id=' + mc.getProperty('uri.var.id') + '/connections';
 			} else if(publicProfileUrl != null && publicProfileUrl != "") {
-				var encoded_publicProfileUrl = encodeURIComponent(mc.getProperty('uri.var.publicProfileUrl'));
+				var encoded_publicProfileUrl = mc.getProperty('uri.var.publicProfileUrl');
```
Comment (prabathabey) [inline linkedin/linkedin-connector/linkedin-connector-1.0.0/org.wso2.carbon.connector/src/main/resources/linkedin_people/getConnections.xml]: Rename variable "encoded_publicProfileUrl" with an appropriate equivalent using CamelCase.

File: linkedin/linkedin-connector/linkedin-connector-1.0.0/org.wso2.carbon.connector/src/test/java/org/wso2/carbon/connector/integration/test/common/ConnectorIntegrationUtil.java [hunk 3]
```diff
@@ -1,25 +1,6 @@
```
Comment (prabathabey) [inline linkedin/linkedin-connector/linkedin-connector-1.0.0/org.wso2.carbon.connector/src/test/java/org/wso2/carbon/connector/integration/test/common/ConnectorIntegrationUtil.java]: License header is missing.

File: formstack/formstack-connector/formstack-connector-1.0.0/org.wso2.carbon.connector/pom.xml [hunk 4]
```diff
@@ -6,7 +6,7 @@
         <groupId>org.wso2.carbon</groupId>
```
Comment (prabathabey) [inline formstack/formstack-connector/formstack-connector-1.0.0/org.wso2.carbon.connector/pom.xml]: License header is missing.

File: linkedin/linkedin-connector/linkedin-connector-1.0.0/org.wso2.carbon.connector/pom.xml (hunk 1 above)
Comment (katheesR) [inline linkedin/linkedin-connector/linkedin-connector-1.0.0/org.wso2.carbon.connector/pom.xml]: Fixed in the PR - https://github.com/wso2/esb-connectors/pull/340

File: formstack/formstack-connector/formstack-connector-1.0.0/org.wso2.carbon.connector/pom.xml (hunk 4 above)
Comment (katheesR) [inline formstack/formstack-connector/formstack-connector-1.0.0/org.wso2.carbon.connector/pom.xml]: Fixed in the PR - https://github.com/wso2/esb-connectors/pull/340

File: linkedin/linkedin-connector/linkedin-connector-1.0.0/org.wso2.carbon.connector/src/test/java/org/wso2/carbon/connector/integration/test/common/ConnectorIntegrationUtil.java (hunk 3 above)
Comment (katheesR) [inline linkedin/linkedin-connector/linkedin-connector-1.0.0/org.wso2.carbon.connector/src/test/java/org/wso2/carbon/connector/integration/test/common/ConnectorIntegrationUtil.java]: Fixed in the PR - https://github.com/wso2/esb-connectors/pull/340

File: linkedin/linkedin-connector/linkedin-connector-1.0.0/org.wso2.carbon.connector/src/main/resources/linkedin_people/getConnections.xml (hunk 2 above)
Comment (katheesR) [inline linkedin/linkedin-connector/linkedin-connector-1.0.0/org.wso2.carbon.connector/src/main/resources/linkedin_people/getConnections.xml]: Recently we have rewritten the LinkedIn connector. The getConnections method is deprecated from the connector

**Pull request merged**